src/sagas.js

```diff
@@ -138,6 +139,7 @@ function* fetchAllVms (action) {
     yield put(removeMissingVms({ vmIdsToPreserve }))
 
     yield put(updateVms({ vms: internalVms, copySubResources: true }))
+    yield fetchVmsSessions({ vms: internalVms })
```

Comment:
This is an additional query per each VM on every refresh cycle which is too high price for too small gain I'd say. 
I think we should do it a bit differently. Maybe something like this:
- do this if "shallowFetch == false" and than showing a msg in the VM details
- also, do this as a part of the attempt to connect to the console. So, the flow would be like this:
  - click connect to console button
  - first check the active sessions, if there is some active session, warn the user that there is an active session and ask if he wants to continue anyway
 - continue with the taking the console normally in case the user confirms

@mareklibra what you think?